runtime/cmake/caches/linux_ppc-64_cmprssptrs_le_gcc.cmake

```diff
@@ -0,0 +1,301 @@
+set(J9VM_INTERP_USE_UNSAFE_HELPER ON CACHE BOOL "")
+set(J9VM_INTERP_VERBOSE ON CACHE BOOL "")
+set(J9VM_IVE_JXE_FILE_RELOCATOR ON CACHE BOOL "")
+set(J9VM_IVE_JXE_IN_PLACE_RELOCATOR ON CACHE BOOL "")
+set(J9VM_IVE_JXE_NATIVES ON CACHE BOOL "")
+set(J9VM_IVE_JXE_OERELOCATOR ON CACHE BOOL "")
+set(J9VM_IVE_JXE_STREAMING_RELOCATOR ON CACHE BOOL "")
+set(J9VM_IVE_ROM_IMAGE_HELPERS ON CACHE BOOL "")
+set(J9VM_JIT_C_HELPERS ON CACHE BOOL "")
+set(J9VM_JIT_CLASS_UNLOAD_RWMONITOR ON CACHE BOOL "")
+set(J9VM_JIT_DYNAMIC_LOOP_TRANSFER ON CACHE BOOL "")
+set(J9VM_JIT_FULL_SPEED_DEBUG ON CACHE BOOL "")
+set(J9VM_JIT_GC_ON_RESOLVE_SUPPORT ON CACHE BOOL "")
+set(J9VM_JIT_NEW_DUAL_HELPERS ON CACHE BOOL "")
+set(J9VM_JIT_NEW_INSTANCE_PROTOTYPE ON CACHE BOOL "")
+set(J9VM_JIT_SUPPORTS_DIRECT_JNI ON CACHE BOOL "")
+set(J9VM_OPT_ANNOTATIONS ON CACHE BOOL "")
+set(J9VM_OPT_BIG_INTEGER ON CACHE BOOL "")
+set(J9VM_OPT_CUDA ON CACHE BOOL "")
+set(J9VM_OPT_DEBUG_INFO_SERVER ON CACHE BOOL "")
+set(J9VM_OPT_DEBUG_JSR45_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_DEPRECATED_METHODS ON CACHE BOOL "")
+set(J9VM_OPT_DYNAMIC_LOAD_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_FRAGMENT_RAM_CLASSES ON CACHE BOOL "")
+set(J9VM_OPT_HARMONY ON CACHE BOOL "")
+set(J9VM_OPT_INLINE_JSRS ON CACHE BOOL "")
+set(J9VM_OPT_INVARIANT_INTERNING ON CACHE BOOL "")
+set(J9VM_OPT_JVMTI ON CACHE BOOL "")
+set(J9VM_OPT_JXE_LOAD_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_MEMORY_CHECK_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_METHOD_HANDLE ON CACHE BOOL "")
+set(J9VM_OPT_MODULE ON CACHE BOOL "")
+set(J9VM_OPT_MULTI_VM ON CACHE BOOL "")
+set(J9VM_OPT_NEW_OBJECT_HASH ON CACHE BOOL "")
+set(J9VM_OPT_NEW_ROM_CLASS_BUILDER ON CACHE BOOL "")
+set(J9VM_OPT_REFLECT ON CACHE BOOL "")
+set(J9VM_OPT_REMOTE_CONSOLE_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_ROM_IMAGE_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_SHARED_CLASSES ON CACHE BOOL "")
+set(J9VM_OPT_SIDECAR ON CACHE BOOL "")
+set(J9VM_OPT_SRP_AVL_TREE_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_STRING_COMPRESSION ON CACHE BOOL "")
+set(J9VM_OPT_SWITCH_STACKS_FOR_SIGNAL_HANDLER OFF CACHE BOOL "")
+set(J9VM_OPT_USE_FFI ON CACHE BOOL "")
+set(J9VM_OPT_USE_FFI_ONLY ON CACHE BOOL "")
+set(J9VM_OPT_VM_LOCAL_STORAGE ON CACHE BOOL "")
+set(J9VM_OPT_ZIP_SUPPORT ON CACHE BOOL "")
+set(J9VM_OPT_ZLIB_COMPRESSION ON CACHE BOOL "")
+set(J9VM_OPT_ZLIB_SUPPORT ON CACHE BOOL "")
+set(J9VM_PORT_OMRSIG_SUPPORT ON CACHE BOOL "")
+set(J9VM_PORT_SIGNAL_SUPPORT ON CACHE BOOL "")
+set(J9VM_PORT_RUNTIME_INSTRUMENTATION ON CACHE BOOL "")
+set(J9VM_PROF_EVENT_REPORTING ON CACHE BOOL "")
+set(J9VM_RAS_DUMP_AGENTS ON CACHE BOOL "")
+set(J9VM_RAS_EYECATCHERS ON CACHE BOOL "")
+set(J9VM_SIZE_OPTIMIZE_SEND_TARGETS ON CACHE BOOL "")
+set(J9VM_THR_ASYNC_NAME_UPDATE ON CACHE BOOL "")
+set(J9VM_THR_LOCK_NURSERY ON CACHE BOOL "")
+set(J9VM_THR_LOCK_RESERVATION ON CACHE BOOL "")
+set(J9VM_THR_PREEMPTIVE ON CACHE BOOL "")
+set(J9VM_THR_SMART_DEFLATION ON CACHE BOOL "")
```

Comment:
A lot of these flags are likely identical across all platforms.  Is there a way to make a common cache file that platforms can share?

We recently went down the copy/paste route in our jenkins build configs and have regretted not commoning the code immediately.  It's now lots of work to undo the decision to duplicate. 

I'd like to avoid that here if we can.

Reply:
If we want to change down the line, it would be a trivial change. My thinking was that once I have all (or nearly all) the platforms up it would be easier to find the common options, and just make those the defaults. However, I'm happy to do some of that refactoring now

Reply:
My preference is to do some of that work upfront.  Too easy to live with the cut/paste forever otherwise.